sonarapp/Program.cs

```diff
@@ -20,6 +20,9 @@ static void Main(string[] args)
 
             Boolean whatifsssfff = (new verify()).verifywhat("whatififfsssf");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'whatifsssfff' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)